deliverystream/aws-kinesisfirehose-deliverystream.json

```diff
@@ -456,6 +499,15 @@
         }
       }
     },
+    "RetryOptions": {
```

Comment:
We can probably rename this to `HttpEndpointRetryOptions` to keep it consistent with the existing naming convention(`RedshiftRetryOptions`|`ElasticsearchRetryOptions`) and this would also make it consistent with the [API](https://docs.aws.amazon.com/firehose/latest/APIReference/API_HttpEndpointRetryOptions.html). Thoughts?

Reply:
Somehow this is launched, not able to change. Original thought is the retry option is kind of same for all destination, probably can reuse the same one for later destination